diff --git a/node_modules/@hapi/oppsy/lib/network.js b/node_modules/@hapi/oppsy/lib/network.js
--- a/node_modules/@hapi/oppsy/lib/network.js
+++ b/node_modules/@hapi/oppsy/lib/network.js
@@ -56,6 +56,7 @@
             }

             if (statusCode) {
+                this._requests[port] = this._requests[port] || { total: 0, disconnects: 0, statusCodes: {} };
                 this._requests[port].statusCodes[statusCode] = this._requests[port].statusCodes[statusCode] || 0;
                 this._requests[port].statusCodes[statusCode]++;
             }
